### ISSUE TITLE
Feature request: creating torrents

### ISSUE THREAD

**Issue by rr-** [rndusr/stig#36]
It would be cool if stig supported creating new .torrent files via a CLI command, for example:

    stig create -o asd.torrent -i ./asd/*.dat --chunk-size 256K --tracker example.com

To keep things simple, it could just create the .torrent file in the designated path, then the user would have to add it manually (possibly supplying the path to the data and manually issuing a recheck).
The paths put into the torrent could be relative to the CWD for maximum flexibility.
`--chunk-size` could be parsed with [`humanfriendly`](https://pypi.python.org/pypi/humanfriendly).

**Comment by rndusr** [comment]
No need for depending on humanfriendly, there's already a Number class in the
utils module. (Wish I knew about humanfriendly before I wrote that.)

I'd add an `--add` option to the `create` command that would automatically add
the new torrent after creating it.

Do you know of a good module that creates torrent files?

**Comment by rr-** [comment]
>I'd add an `--add` option to the `create` command that would automatically add the new torrent after creating it.

That's pretty cool.

>Do you know of a good module that creates torrent files?

I don't know of any such module, unfortunately.

**Comment by sometoby** [comment]
>Do you know of a good module that creates torrent files?

I've been using [dottorrent](https://github.com/kz26/dottorrent) to create my torrent files, maybe that's worth a look.

**Comment by rndusr** [comment]
Looks like that's perfect. Thanks!

**Comment by rndusr** [comment]
I've added the command 'create'.

It doesn't have the --add option I proposed because I don't think it's useful. You can just run `stig create path/to/content -f path/to/my.torrent and add path/to/my.torrent`.

Feedback is welcome.

**Comment by rndusr** [comment]
I've decided to move the ability to create torrents into a separate project.

There is almost no common code between *interfacing with a torrent client* and
*creating a torrent file*. That makes the 'create' command almost feel like a
foreign object. Also, by moving it in a separate project, it can be used by
people who don't need a transmission-daemon client without lots of overhead
code.

The new project is here: https://github.com/rndusr/torf-cli

I've also implemented a torrent library: https://github.com/rndusr/torf
